lua/telescope/_extensions/file_browser/picker.lua

```diff
@@ -54,6 +54,7 @@ local fb_picker = {}
 ---@field respect_gitignore boolean: induces slow-down w/ plenary finder (default: false, true if `fd` available)
 ---@field browse_files function: custom override for the file browser (default: |fb_finders.browse_files|)
 ---@field browse_folders function: custom override for the folder browser (default: |fb_finders.browse_folders|)
+---@field hide_parent_entry boolean: hide `../` in the file browser (default: false)
```

Comment:
Let's call this `hide_parent_dir`.

It's a bit clearer as not everyone is familiar with telescope terminology.